yt/loaders.py

```diff
@@ -1457,3 +1461,46 @@ def load_sample(
         loadable_path = loadable_path.joinpath(load_name, specific_file)
 
     return load(loadable_path, **kwargs)
+
+
+# --- Loader for tar-based datasets ---
+def load_archive(
+    fn: str, path: str, ratarmount_kwa: dict = None, *args, **kwargs
+) -> Dataset:
+    if ratarmount_kwa is None:
+        ratarmount_kwa = {}
+
+    from yt.utilities.on_demand_imports import _ratarmount
```

Comment:
the way on_demand_imports work, it doesn't cost anything to put this at the top level